src/test/build/shared-build-targets-utils/Utils/DotNetCli.cs

```diff
@@ -32,7 +32,8 @@ public Command Exec(string command, params string[] args)
                 newArgs.Insert(0, "-v");
             }
 
-            return Command.Create(Path.Combine(BinPath, $"dotnet{Constants.ExeSuffix}"), newArgs);
+            return Command.Create(Path.Combine(BinPath, $"dotnet{Constants.ExeSuffix}"), newArgs)
+                .EnvironmentVariable("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "1");
```

Comment:
Yes, the environment variable is the only way to control this in the CLI.  There's been asks since before 1.0.0-preview1 to have a CLI .config file, which would control things like this.  But that feature's priority has never bubbled up high enough to gain attention.

/cc @livarcocc 

> It seems like we should avoid this in any cases where we are using an xcopy CLI.

For sure we should avoid it in any CI system.  This is yet another reason to build a "minimal .NET Core SDK" that doesn't have things like this in it. @Eilon and I were just discussing that.

I've asked @chcosta to think about a more "global" way of setting this env var in our build, so it is always on, no matter if the build is invoked on dev machine, Jenkins or VSO.